common:
  gait: trot
  # Use local planner with or without elevation_mapping
  blind: true
  # Use cheater odometry from simulation
  perfect_odometry: true
  sim_slowdown: 1.0

high_command:
  use_move_base: false
  velocity_x: 0.2
  velocity_y: 0.0
  velocity_yaw: 0.0
  foot_height: 0.08 
  body_height: 0.27 

foot_planning:
  foothold_search_radius: 0.1
  # Update footholds during the whole swing phase
  upd_footholds_continuously: false

mpc:
  mpc_step_dt: 0.02
  mpc_horizon_steps: 15

body_planner:
  # Stairs
  height_lpf_freq: 30.0
  pitch_lpf_freq: 10.0

  # Sparse stepping stones
  # height_lpf_freq: 0.9
  # pitch_lpf_freq: 0.9
